Dhruwang/formbricks
# feat: implement reset confirmation dialog for Formbricks attributes

---

### Pull Request Title
`feat: implement reset confirmation dialog for Formbricks attributes`

## What does this PR do?
This PR implements a confirmation dialog that prompts users to confirm before resetting attributes in Formbricks. This enhancement aims to prevent accidental resets and improve user experience. The confirmation dialog appears when the "Reset" button is clicked, ensuring users are aware of the action's consequences.

**Fixes**: #123 (replace with your issue number)

## How should this be tested?
- **Test A**: Click the "Reset" button without confirming to ensure attributes are not reset.
- **Test B**: Click the "Reset" button and confirm to verify that attributes are successfully reset.

## Checklist

### Required
- [x] First PR at Formbricks? [Please sign the CLA!](https://cla-assistant.io/formbricks/formbricks) Without it we won't be able to merge it 🙏

### Appreciated

---
